backend/tests/unit/graphql/queries/test_task.py

```diff
@@ -85,6 +85,42 @@
 """
 
 
+QUERY_TASK_WITH_LOG_OFFSET = """
+query TaskQuery(
+    $related_nodes: [String]
+) {
+  InfrahubTask(related_node__ids: $related_nodes, log_offset: 1, log_limit: 10) {
+    count
+    edges {
+      node {
+        conclusion
+        created_at
+        id
+        related_node
+        related_node_kind
+        related_nodes {
+            id
+            kind
+        }
+        title
+        updated_at
+        logs {
```

Comment:
Could we move `limit` and `offset` here instead of using `log_limit` and `log_offset` ?